mod/deckorate_attribution/assets/style/reference.scss

```diff
@@ -1,7 +1,36 @@
 .TYPE-reference .RIGHT-Xsubject.card-editor {
-  display: none
+  display: none;
 }
 
-.attribution-alert-link {
-  font-size: small;
+.TYPE-reference .RIGHT-adaptation.card-editor {
+  .form-check {
+    appearance: none;
+    background-clip: padding-box;
+    background-color: #fff;
+    border: 1px solid #ced4da;
+    border-radius: 50.4rem;
+    color: #34343b;
+    flex: 1;              
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.5;
+    min-height: 1.5rem;
+    margin-right: 1rem; 
+    padding: 0.375rem 1rem;
+    padding-left: 2.2rem;
+    text-align: left;
+    transition: border-color 0.15s ease-in-out,box-shadow 0.15s ease-in-out;
+  }
+  
+  .form-check:last-child {
+    margin-right: 0;
+  }
```

Comment:
fyi, another handy shortcut (which you don't have to use) is to use `&` to add on to the current selector. Eg you can do the following:
```
.form-check {
  appearance: none;
  ...
  &:last-child {
    margin-right: 0;
  }
  
}
```